apps/web/src/pages/integrations/IntegrationsStorePage.tsx

```diff
@@ -58,79 +58,78 @@ export function IntegrationsStore() {
     }
   }
 
+  if (isLoading) {
+    return null;
+  }
```

Comment:
moved this statement to outside

Reply:
Maybe we should show a spinner or something if it takes long time?

Reply:
sure, we can have a loading overlay that we do show everywhere currently :)